subscribed_topics:
  thermal_msg_topic: /flir_lepton_hardware_interface/flir/msg

published_topics:
  thermal_poi_topic: thermal/poi
